mudlet-mapper.xml

```diff
@@ -7346,7 +7349,7 @@ end
 function mmp.echonums(roomname, area)
   local t = mmp.searchRoomExact(roomname)
   if area then
-    t = filterRooms(t, area)
+    t = mmp.filterRooms(t, area)
```

Comment:
There is a local function at https://github.com/IRE-Mudlet-Mapping/ire-mapping-script/blob/424e11d74fbca728d2fb6146700001027d6c0858/mudlet-mapper.xml#L7333 though, is not working?

Reply:
Hmm. I was getting errors on veiled farsee, prompted me to go through and replace all local calls with the correct global call. The global function is relatively new.

Reply:
Ah ok, then the local one should be deleted so we don't get confused